docs/section12/Rule12-9.md

```diff
@@ -1,10 +1,10 @@
 
-# Receptacle - Rule 12-9
+# Receptacle - Rule 12-9  
 
 **Rule ID:** 12-9  
 **Rule Description:** User RMR Receptacle Schedule in Baseline RMR?  
 **Rule Assertion:** Baseline RMR = User RMR  
```

Comment:
I think our default approach to such rules is to check that P-RMR=U-RMR, and then that B-RMR is estalbished correctly relative to P-RMR, which in this case is B-RMR=P-RMR. This breakes with this tradition by checking that B-RMR=U-RMR. I suggest keeping it consistent.

I am also not sure what the actual rule is here. There are instances when B-RMR equipment schedule is allowed to differe from P-RMR schedule. More on this below.